libs/game/menu.ts

```diff
@@ -1166,7 +1216,9 @@ namespace menu {
                 .to(screen.width - this.margin)
                 .duration(200)
                 .onEnded(() => {
+                    console.log(`show list`)
```

Comment:
Looks like you're using `log()` instead of `console.log()` everywhere else

Reply:
Yes, it is a menu specific log that I can turn on/off